### PR TITLE
Fixing meta

### PR DESCRIPTION
**Pull request by tjayrush**

### CODE REVIEW

File: src/apps/chifra/pkg/blockRange/range.go [hunk 1]
```diff
@@ -73,6 +74,14 @@ type BlockRange struct {
 	Modifier     Modifier
 }
 
+func (br BlockRange) String() string {
```
Comment (dszlachta) [inline src/apps/chifra/pkg/blockRange/range.go]: This is actually producing a JSON string, right? Maybe we want to call this function `MarshallJSON` and keep `String` for a potential future use (e.g. at some point we may want to serialize BlockRange to `0-10000`)

**Review by dszlachta — APPROVED** [review]

File: src/apps/chifra/pkg/blockRange/range.go (hunk 1 above)
Comment (tjayrush) [reply src/apps/chifra/pkg/blockRange/range.go]: Changing now.

Comment (tjayrush) [reply src/apps/chifra/pkg/blockRange/range.go]: Should it take a pointer to the object or the object itself?

**Comment by dszlachta** [comment]
It can be a pointer, since 1) you don’t modify the object, 2) ranges are not large
See https://go.dev/doc/faq#methods_on_values_or_pointers

Dawid Szlachta

**Comment by tjayrush** [comment]
> 

This helps: "Whether to define the receiver as a value or as a pointer is the same question, then, as whether a function argument should be a value or a pointer."

But in C++, I would use a constant reference as a parameter in this case. I don't think there is such as thing as a constant in Go. Is there?

**Comment by tjayrush** [comment]
This helps too: "Next is consistency. If some of the methods of the type must have pointer receivers, the rest should too, so the method set is consistent regardless of how the type is used"

**Comment by dszlachta** [comment]
No, there’s no such constants in Go

Dawid Szlachta

**Comment by tjayrush** [comment]
You can merge this if you think it's okay.

**Pull request merged**